### DISCUSSION TITLE
Change default port 3000

### DISCUSSION THREAD

**Discussion by JNNelson** [chatwoot/chatwoot#9723] [Q&A]
Sorry for the question, it may seem obvious, but I didn't find where I can change the default port 3000 to something else in the .ENV file.
Or is this not possible?

**Comment by MarcioLacerda** [comment]
Não sei se existe uma variável para ser definida no .ENV, mas exite o parâmetro -p <porta> que vc pode usar no comando que sobe o serviço.

Eu uso docker e coloco o seguinte no command:

`command: bundle exec rails s -p 3030`

**Comment by JNNelson** [comment]
Ola @MarcioLacerda .
Minha instalacao esta fora do Docker, mas sua resposta acredito ser util para quem usa este ambiente.
Normalmente é no arquivo .ENV que vai esta informacao, mas a documentacao do Chatwoot (https://www.chatwoot.com/docs/self-hosted/configuration/environment-variables/) nao consta nenhuma referencia.
Nem mesmo no utilitario cwctl na linha de comando tem a opcao de modificar a porta.
